pages/common/tldr.md

```diff
@@ -1,20 +1,20 @@
 # tldr
 
-> Displays simple help pages for command-line tools, from the tldr-pages project.
+> Display simple help pages for command-line tools from the tldr-pages project.
 > More information: <https://tldr.sh>.
 
-- Show the tldr page for a command (hint: this is how you got here!):
+- Print the tldr page for a specific command (hint: this is how you got here!):
 
 `tldr {{command}}`
 
-- Show the tldr page for `cd`, overriding the default platform:
+- Print the tldr page for a specific subcommand:
 
-`tldr -p {{android|linux|osx|sunos|windows}} {{cd}}`
+`tldr {{command}}-{{subcommand}}`
 
-- Show the tldr page for a subcommand:
+- Print the tldr page for a command for a specific platform:
 
-`tldr {{git-checkout}}`
+`tldr {{-p|--platform}} {{android|linux|osx|sunos|windows}} {{command}}`
 
-- Update local pages (if the client supports caching):
+- Print the tldr page for a command in a specific language:
```

Comment:
but maybe the client doesn't implement cache :man_shrugging:

Reply:
also this description doesn't match the command at all